transformations/transformations.js

```diff
@@ -2,7 +2,7 @@ import { HOST_URL } from '../yasp.config';
 import moment from 'moment';
 
 // TODO move to utility module
-const isRadiant = (p) => p.player_slot < 128;
+const isRadiant = (playerSlot) => playerSlot < 128;
```

Comment:
The way I changed this before I was passing the specific value, which meant there was an error with it. This fixes the error.